slack-base/src/main/java/com/hubspot/slack/client/methods/params/migration/MigrationExchangeParamsIF.java

```diff
@@ -0,0 +1,21 @@
+package com.hubspot.slack.client.methods.params.migration;
+
+import java.util.List;
+import java.util.Optional;
+
+import org.immutables.value.Value;
+
+import com.fasterxml.jackson.databind.PropertyNamingStrategy.SnakeCaseStrategy;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import com.hubspot.immutables.style.HubSpotStyle;
+
+@Value.Immutable
+@HubSpotStyle
+@JsonNaming(SnakeCaseStrategy.class)
+public interface MigrationExchangeParamsIF {
+  List<String> getUsers();
```

Comment:
Does it make sense to have this as a `Set`?

Reply:
The method can be renamed to `getUserIds` to better reveal the intention. You can add `@JsonProperty("users")` then to make sure it's compatible with the API.

Reply:
Yes makes sense. Let me do that changes.

Reply:
Updated.